.github/workflows/deploy-and-lighthouse.yml

```diff
@@ -26,8 +26,8 @@ jobs:
         env:
           COMMIT_SHA: ${{ github.sha }}
 
-      - name: Deploy to now
-        run: npx now --confirm --token $NOW_TOKEN --meta commit=${GITHUB_SHA} --meta branch=${GITHUB_REF} --scope frontity | tee >(grep '^https:\/\/frontity-org-' > deploy-url.txt)
+      - name: Deploy to vercel
+        run: npx vercel --confirm --token $NOW_TOKEN --meta commit=${GITHUB_SHA} --meta branch=${GITHUB_REF} --scope frontity | tee >(grep '^https:\/\/frontity-org-' > deploy-url.txt)
         env:
           NOW_TOKEN: ${{ secrets.NOW_TOKEN }}
```

Comment:
Would it make sense to change also this env variable to something like `VERCEL_TOKEN`?

Reply:
I don't know how to rename a secret, so I'm just going to leave that for now.